project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/cache/ColorCache.java

```diff
@@ -0,0 +1,82 @@
+package com.samsung.sec.dexter.eclipse.ui.cache;
+
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.graphics.RGB;
+import org.eclipse.swt.widgets.Display;
+
+/**
+ * Class for caching colors
+ * 
+ * @author Emil
+ *
+ */
+public final class ColorCache {
```

Comment:
If this is a copy of open source codes, please check the license first.
If this code is not eligible for us, replace this code as yours.
And if necessary, add license text.